Rakefile

```diff
@@ -6,4 +6,4 @@ require File.expand_path('../config/application', __FILE__)
 
 Support::Application.load_tasks
 
-task :default => [:spec, 'jasmine:ci']
+task :default => [:spec, 'jasmine:ci', :lint]
```

Comment:
Does it run in this order? if so, would it make sense for it to do lint first so it fails before running tests?

Reply:
It doesn't seem to run in this order.

Reply:
Managed to make it run in the order we want by clearing the default rake task.

Reply:
Great 👍